src/libNode/DSBlockProcessing.cpp

```diff
@@ -193,6 +180,10 @@ bool Node::VerifyDSBlockCoSignature(const DSBlock& dsblock)
         == false)
     {
         LOG_GENERAL(WARNING, "Cosig verification failed");
+        for (auto& kv : keys)
+        {
+            LOG_GENERAL(WARNING, "" << kv);
```

Comment:
`"" << kv` looks unnecessary, any specific reason for doing this?

Reply:
This is to identify who cosign the message. This is important for debugging and such information should be captured when cosig verification fail.

Reply:
Sorry for not making the question clear, what I mean is can we just use `kv` instead of `"" << kv`?